server/conf/application.conf

```diff
@@ -14,6 +14,8 @@
 # And if an environment variable exists when there is no other subsitution, then
 # HOCON will fall back to substituting environment variable:
 #mykey = ${JAVA_HOME}
+#
+# Environment variable substitutions should all be SCREAMING_SNAKE_CASE.
```

Comment:
I didn't know it was called SCREAMING_SNAKE_CASE. Made me laugh!

Reply:
:)